### PR TITLE
Google Sign-in support (not activated)

### PR DESCRIPTION
**Pull request by spolu**

### CODE REVIEW

File: front/pages/api/auth/[...nextauth].js [hunk 1]
```diff
@@ -8,14 +8,17 @@ export const authOptions = {
...
-    // }),
+    GoogleProvider({
+      clientId: process.env.GOOGLE_CLIENT_ID,
+      clientSecret: process.env.GOOGLE_CLIENT_SECRET,
+    }),
   ],
   secret: process.env.NEXTAUTH_SECRET,
   callbacks: {
     session: async ({ session, token, user }) => {
+      // console.log("TOKEN", token);
```
Comment (fontanierh) [inline front/pages/api/auth/[...nextauth].js:19]: nit 🧹

**Review by fontanierh — COMMENTED** [review]
LGTM

**Review by fontanierh — DISMISSED** [review]

File: front/pages/api/auth/[...nextauth].js (hunk 1 above)
Comment (spolu) [reply front/pages/api/auth/[...nextauth].js:19]: I keep them around as I they are useful for debugging when working with auth providers.

**Comment by spolu** [comment]
(pushed fix to linter, will merge as is)

**Pull request merged**